report-ui/style.css

```
@@ -860,14 +1069,30 @@ select:focus {
+  padding: 12px 14px;
   border: 1px solid var(--border-color-strong);
-  background: var(--bg-card);
-  border-radius: 10px;
-  box-shadow: 0 8px 24px rgba(0, 0, 0, 0.28);
+  background: rgba(
+    15,
+    23,
+    42,
+    0.85
+  ); /* Slightly more transparent for glass effect */
+  backdrop-filter: blur(16px);
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)